nbs/docs/models/GARCH.ipynb

```diff
@@ -233,6 +233,26 @@
     "rcParams['figure.figsize'] = (18,7)"
```

Comment:
<p>Please remove this cell</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/statsforecast/pull/925/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>